cli/cluster/membership.go

```diff
@@ -50,17 +62,22 @@ func GetMembershipInstances(instancesConf *InstancesConf, ctx *context.Ctx) (*Me
 		return nil, fmt.Errorf("failed to get membership instances: %s", err)
 	}
 
+	for _, filter := range filters {
+		filter(membershipInstances, runningInstances)
+	}
+
 	return membershipInstances, nil
 }
 
-func ConnectToMembership(conn *connector.Conn, runningInstancesNames []string, instancesConf *InstancesConf) error {
-	// probe all instances mentioned in topology
+func ConnectToMembership(conn *connector.Conn, runningInstancesNames map[string]string,
+	instancesConf *InstancesConf) error {
+	// probe all running instances mentioned in topology
 	var urisToProbe []string
 
 	for _, instanceName := range runningInstancesNames {
 		instanceConf, found := (*instancesConf)[instanceName]
 		if !found {
-			return fmt.Errorf("Instance %s  isn't found in instances config", instanceName)
+			return fmt.Errorf("Instance %s isn't found in instances config", instanceName)
```

Comment:
In general, I don't mind fixing some stuff while working on a patch, but it's better to do it in separate commits.
https://github.com/tarantool/tarantool/wiki/Code-review-procedure#general-coding-points-to-check

> `Don't do a change if it is not related to the patch...`

Reply:
So, should I open a separate PR for removing this extra space? :)

Reply:
> So, should I open a separate PR for removing this extra space? :)

OMG, NO!

Reply:
If I make several such edits in PR - I just squash them into one common (like `syntax fix`) commit or something like that. And they don't interfere reviewer looking at the rest code.